### DISCUSSION TITLE
Lock down root login

### DISCUSSION THREAD

**Discussion by questautomata** [minio/minio#20741] [Q&A]
is it possible to lock down the default root user/login? i created a new user with limited access for myself to log into, but couldn't someone just brute force the user/password? im pretty new when it comes to devops and security. my goal currently is to use authentik as an MFA provider so i can somehow require the root user to authenticate via MFA

**Comment by questautomata** [comment]
i know you can turn off root user (is it just for api and not console?) - but i would rather lock the login behind an MFA if possible

**Comment by marktheunissen** [comment]
> is it possible to lock down the default root user/login? 

https://min.io/docs/minio/linux/reference/minio-server/settings/root-credentials.html#envvar.MINIO_API_ROOT_ACCESS

> is it just for api and not console?

For both

> my goal currently is to use authentik as an MFA provider

Disable root access then use your provider
